packages/liquidsoap/liquidsoap.2.0.1/opam

```diff
@@ -36,13 +36,7 @@ depopts: [
   "dssi"
   "faad"
   "fdkaac"
-  "ffmpeg-avutil"
-  "ffmpeg-av"
-  "ffmpeg-avcodec"
-  "ffmpeg-avdevice"
-  "ffmpeg-avfilter"
-  "ffmpeg-swresample"
-  "ffmpeg-swscale"
```

Comment:
According to https://github.com/savonet/liquidsoap/commit/d21649a6c3594dbae8a702b99b53fb52163d3401 2.0.1 still has optional dependencies toward these packages.

This does not seem right. For example this unlocks opam to compile both ffmpeg-av at the same time as liquidsoap, which could cause random failures/race-conditions.

Reply:
I see you point. Yes, the underlying problem is that the build system for liquidsoap `2.0.x` makes the wrong assumptions about which compbinations of ffmpeg subpackages it needs for its files.. 

I have fixed that in the upcoming `2.0.2` bugfix release, which directly depends on the top-level `ffmpeg` library. It will come out as soon as this package is accepted, after which I believe these issues will matter much less.

Reply:
In the future, it might be easier to split the package/libraries into one per optional dependency that you have right now and transform the liquidsoap package into a package providing the binary (with optional dependencies to the sublibraries but with `= version` constraints).
Opam isn’t really clever enough to handle the truck load of issues that comes with optional dependencies and it might be easier for your users.

Reply:
Definitely! This is part of the plan with the `dune` migration but, liquidsoap is a large codebase with historically a lot of finer-tunes compilation details.